src/sentry/api/endpoints/organization_event_details.py

```diff
@@ -91,7 +91,7 @@ def oldest_event_id(self, snuba_args, event):
 
     def _get_filter(self, snuba_args):
         return eventstore.Filter(
-            conditions=snuba_args["conditions"],
+            conditions=snuba_args["conditions"] + [["type", "!=", "transaction"]],
```

Comment:
Don't we need to get transaction events when viewing them?

Reply:
Oh.... transaction events also use this endpoint? In that case this won't work

Reply:
Removed the condition from this endpoint, thanks

Reply:
Yes this endpoint is used to get the payload to render the details modal.